app/models/run.rb

```diff
@@ -218,6 +218,12 @@ def possible_timesave(timing)
     duration(timing) - sum_of_best(timing)
   end
 
+  def video
+    return nil unless video_url
```

Comment:
@glacials Do you think it's worth it to just pass in `nil` to video like how `Duration` handles it?  Since we aren't printing anything it seems pointless, but having things be consistent is also nice sometimes.

Reply:
I'll take a look at `Duration`. Consistency is important.

Reply:
Yeah it might be worth trying it out to see how it feels if it's not too much work. For reference this is called the [null object pattern][1]. I've been really liking `Duration` ever since we made that change. However if doing this becomes more complicated alongside my other comment re: making Video a proper ActiveRecord model, then don't worry about it for now.

[1]: https://en.wikipedia.org/wiki/Null_object_pattern

Reply:
The current version does take `nil` and no longer does this check. But it sounds like things are moving to a real AR model, so this will go away.